README.md

```diff
@@ -42,6 +42,30 @@ the [plugins repository] or in the [Lite XL plugins repository].
 Additional color themes can be found in the [colors repository].
 These color themes are bundled with all releases of Lite XL by default.
 
+## Quick Build Guide
+
+If you compile Lite XL yourself, it is recommended to use the script
+`build-packages.sh`:
+
+```sh
+bash build-packages.sh <arch>
+```
+
+The script will run Meson and create a zip file with the application or,
+for Linux, a tar compressed archive. Lite XL can be easily installed
+by unpacking the archive in any directory of your choice.
+
+Otherwise the following is an example of basic commands if you want to customize
+the build:
+
+```sh
+meson setup --buildtype=release --prefix <prefix> build
+meson compile -C build
+DESTDIR="$(pwd)/lite-xl" meson install --skip-subprojects -C build
+```
+
+where `<prefix>` might be one of `/`, `/usr` or `/opt`, the default is `/usr/local`.
```

Comment:
You may add "Please note that the package is relocatable to any prefix and the option prefix affects only the place where the application is actually installed.

Reply:
the install in this case just copies the files inside a `lite-xl` directory in the source tree, so now I'm not even sure if to add that explanation I wrote about the prefix and just set it to `/`, specifying that Meson should never be used to install a package in the system but to use the proper package manager instead.

Reply:
Lets just give to the user the important information without telling them what they should do or not do. People are smart enough to know better.

What they need to know:

- the prefix and DESTDIR can be used to choose the installation prefix and destination, fine
- the lite-xl executable and data dir are perfectly relocatable to any prefix, independently of the build-time prefix, we just give the information so people know that

Reply:
So smart that they later comes in chat with strange issues after doing the wrong things because they are just developers and no packaging experience, and I see this from at least 10 years, anyway I'll add what you said.